packages/core/src/external/aws/lambda-logic/document-uploader.ts

```diff
@@ -119,22 +124,34 @@ function buildDestinationKey(cxId: string, patientId: string, docId: string): st
   return `${cxId}/${patientId}/${UPLOADS_FOLDER}/${cxId}_${patientId}_${docId}`;
 }
 
-async function createAndUploadMetadataFile(
-  s3Utils: S3Utils,
-  cxId: string,
-  patientId: string,
-  docId: string,
-  hash: string,
-  size: string,
-  docRef: DocumentReference,
-  destinationBucket: string
-): Promise<void> {
+async function createAndUploadMetadataFile({
+  s3Utils,
+  cxId,
+  patientId,
+  docId,
+  hash,
+  size,
+  docRef,
+  s3MetadataFileName,
+  destinationBucket,
+}: {
+  s3Utils: S3Utils;
+  cxId: string;
+  patientId: string;
+  docId: string;
+  hash: string;
+  size: string;
+  docRef: DocumentReference;
+  s3MetadataFileName: string;
+  destinationBucket: string;
+}): Promise<void> {
   const createdTime = new Date().toISOString();
   const uniquePatientId = createPatientUniqueId(cxId, patientId);
   const title = docRef.description;
   const classCode = docRef.type;
   const practiceSettingCode = docRef.context?.practiceSetting;
   const healthcareFacilityTypeCode = docRef.context?.facilityType;
+  console.log(`Creating metadata file for docId: ${docId}`);
```

Comment:
Left-over?

If not and we're going to add a log to the production env, let's make it more useful and also add the patient ID, it helps tracking stuff when debugging 🙏

Reply:
added patientId and cxId